chrome_manifest_v3/class_model.js

```
@@ -0,0 +1,456 @@
+var Gmail2Trello = Gmail2Trello || {}; // must be var to guarantee correct scope
```

Biome (1.9.4)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)